packages/backend/src/connector/core/controllers/stream.ts

```diff
@@ -33,31 +33,33 @@ export function createStreamController(): ILPMiddleware {
     const connectionKey = streamReceivedKey(connectionId)
     // Thanks to Redis's `stringNumbers:true`, `incrby` returns a string rather than a number.
     // This ensures that precision isn't lost when dealing with integers larger than MAX_SAFE_INTEGER.
-    const [[err, totalReceived], [err2]] = await redis
```

Comment:
The types for this redis method aren't great...
Maybe we could keep this destructuring after we've checked for `if (query)`, just to avoid having to index the nested array?

Reply:
I'll see what I can do.